### ISSUE TITLE
Prepare release 3.11.0

### ISSUE THREAD

**Issue by ann0see** [jamulussoftware/jamulus#3255]
**Target timeline**
| Phase | Date |
|--------|--------|
| Scheduled feature freeze / Start of translation process | **In the past** |
| Targeted translation completion date: | 2024-08-26 |
| Approximate release date: | **2024-09-22** |
| Current state: | **Release Candidate 1** | 

(Due to a couple of late changes post freeze, the planned date is moved back three weeks.  Hopefully RC1 by 2024-09-15.)

**Checklist**
- [ ] Assign this issue to the release shepherd who is in charge of managing this checklist.
- [x] Pin this issue
- [x] Ensure that all issues/PR targeted for this release are done by checking the Project board with [the appropriate filter for this release](https://github.com/orgs/jamulussoftware/projects/5/views/5). Remind main developers to review entries in *Waiting on team* state.
- [x] Agree to de-tag unfinished Issues/PRs.
- [x] Declare a freeze for code and website by updating this Issue and adding a comment. PRs can still be worked on and may get reviewed, but must not be merged unless agreed explicitly.
- [x] Check the `needs documentation` label for any outstanding PRs flagged for this release and remove that label if done.
- [x] Check ./Jamulus -h output against the Wiki [Include-Client](https://github.com/jamulussoftware/jamuluswebsite/blob/next-release/wiki/en/Include-Client-Commands.md)/[Server-Commands](https://github.com/jamulussoftware/jamuluswebsite/blob/next-release/wiki/en/Include-Server-Commands.md) pages and [man page (distributions/Jamulus.1)](https://github.com/jamulussoftware/jamulus/blob/main/distributions/Jamulus.1). Update if necessary.
- [x] Start Website translations
  - [x] [Check for broken links](https://github.com/jamulussoftware/jamuluswebsite/blob/release/contribute/en/Release-Process.md#website-check-links)
  - [x] Open a Pull Request from `next-release` to release, set it as "Draft", sanity check for conflicts and any obvious problems.
  - [x] Declare a full freeze of the `next-release` and `release` branch. No changes should be made from now on to ensure translators don't have to work twice.
  - [ ] Check if the list of translators in `tools/create-translation-issues.sh`. Make sure issue text is up-to-date. Add any URLs that will need localisation into the "New/Changed screenshots" section.
  - [x] Unlock Web translations on [Weblate](https://hosted.weblate.org/projects/jamulus/) if needed
  - [x] Publish a new [Weblate announcement](https://hosted.weblate.org/projects/jamulus/#announcement) to notify translators of the translation deadline if not already done for the app.
  - [x] Create a translation issue for each language with `tools/create-translation-issues.sh` using `web` argument (see notes in script).
  - [x] If anyone finds critical issues now, all translators must be made aware of them and all languages should be updated.
- [x] Start App translations
  - [x] Generate `.ts` files in main via `lupdate`
  - [ ] Check if the list of translators in `tools/create-translation-issues.sh` is up-to-date
  - [x] Create a translation issue for each language with `tools/create-translation-issues.sh` using `app` argument.
  - [x] Unlock Translations on [Weblate](https://hosted.weblate.org/projects/jamulus/) if needed
  - [x] Publish a new [Weblate announcement](https://hosted.weblate.org/projects/jamulus/#announcement) to notify translators of the translation deadline if not already done for the Website.
- [x] [Update the Changelog](https://jamulus.io/contribute/Release-Process#updating-the-changelog)
- [x] [Tag a beta release](https://jamulus.io/contribute/Release-Process#steps-for-a-specific-release)
  - [x] Inform emlynmac for signing on macOS, and upload signed binary from [his repo](https://github.com/emlynmac/jamulus/releases/) to [ours](https://github.com/jamulussoftware/jamulus/releases/)
  - [x] [Announce the beta release](https://github.com/orgs/jamulussoftware/discussions/3342) on Github Discussions. Pin the thread.
  - [x] Get feedback on the stability and resource usage (memleaks?, crashes?, installation issues?) of the beta release
- [x] Finish App translations
  - [x] Review translation PRs according to [release process checklist](https://jamulus.io/contribute/Release-Process#3-update-the-ts-files-returned-by-translators)
  - [x] Wait for all PRs to be merged (missing translations will revert to English automatically).
  - [x] Check for conflicting accelerator keys (see `tools/checkkeys.pl`)
  - [ ] [Check for broken links](https://github.com/jamulussoftware/jamuluswebsite/blob/release/contribute/en/Release-Process.md#website-check-links)
- [x] Check the milestone for mergable stuff again
- [x] [Update the Changelog](https://jamulus.io/contribute/Release-Process#updating-the-changelog)
- [x] [Tag a release candidate](https://jamulus.io/contribute/Release-Process#steps-for-a-specific-release) (inform emlynmac for signing on macOS and upload signed binary from his repo to ours).
  - [x] Announce the release candidate on Github Discussions. Pin the thread. Unpin and lock the beta thread.
  - [x] Draft an announcement, include all contributors via `tools/get_release_contributors.py`
- [x] [Update the version number in `Jamulus.pro` and add the release date to the Changelog header and commit](https://jamulus.io/contribute/Release-Process#steps-for-a-specific-release)
- [x] [Update the Changelog](https://jamulus.io/contribute/Release-Process#updating-the-changelog) **MOVE *BEFORE* PREVIOUS STEP**
- [x] Tag this commit as `r3_y_z`
  - [x] Wait for the build to complete
  - [x] Push tag to ann0see/jamulussign for signing on macOS and upload signed binary [from his](https://github.com/ann0see/jamulussign/releases/) repo [to ours](https://github.com/jamulussoftware/jamulus/releases).
  - [x] Do a smoke test for Windows/Mac/Linux -- Do the binaries start/connect properly? Can earlier Jamulus versions properly connect to a server based on the new release?
  - [x] [Force tag that tag as `latest` and push.](https://jamulus.io/contribute/Release-Process#if-this-is-a-proper-release-move-the-latest-tag)
  - [x] [Upload the artifacts to SourceForge and set defaults](https://jamulus.io/contribute/Release-Process#upload-binaries-to-sourceforge).
  - [x] Update download links on the website by editing `_config.yml` in `next-release`
  - [x] Disable branch protection rule of the `release` branch by clicking on "Edit" on the [Branches page](https://github.com/jamulussoftware/jamuluswebsite/settings/branches) and adding a `_` behind `release`.  
  - [x] Publish Website release by **squashing** and merging `next-release` into `release`
  - [x] Enable branch protection rule of the `release` branch after the site and the `.po` files are published by removing the `_` from the branch protection rule you edited on the [Branches page](https://github.com/jamulussoftware/jamuluswebsite/settings/branches).
- [x] Announce the new release with a summary of changes (+ link to the changelog for details) and a link to the download page
    - [x] On Github Discussions in the Announcements section. Lock the announcement thread. Pin the thread. Unpin and lock release candidate thread.
    - [x] On Facebook in the group "Jamulus (official group)". Turn off replies.
- [ ] Trigger the update notification by updating both Update Check Servers with the new version (@pljones for update02, email corrados for update01)
- [x] [Prepare `Jamulus.pro` (`dev` suffix) and ChangeLog (add a header) for the [next release](https://jamulus.io/contribute/Release-Process#make-the-main-branch-ready-for-post-release-development)
- [x] Check that all Issues and PRs tagged for this release are in Done/Closed state.
- [x] Close the release milestone in both jamulus and jamuluswebsite repos
- [x] Create a milestone for the next minor release in jamulus and jamuluswebsite repos
- [ ] Update this template in https://jamulus.io/contribute/Release-Process with any improvements if needed.
- [ ] Unpin and close this issue
- [ ] Determine if a release retrospective is needed, create on Discussions if required

**Comment by ann0see** [comment]
https://github.com/jamulussoftware/jamulus/pull/3309 is the last PR for 3.11.0 before a feature freeze. Is this correct or do we want to bump Qt also?

**Comment by pljones** [comment]
> - [ ] Start App translations
>   - [ ] Generate .ts files in main via lupdate

Is this still needed (the `.ts` files) or is that what's now being done automatically?

**Comment by pljones** [comment]
Oh yes -- Code and Documentation freeze has been in place for a while, all remaining changes should be completed ASAP, please.

**Comment by pljones** [comment]
> #3309 is the last PR for 3.11.0 before a feature freeze. Is this correct or do we want to bump Qt also?

(Somewhat later...) Code is complete, the only outstanding PRs are website:
* [jamulus/jamuluswebsite#989](https://github.com/jamulussoftware/jamuluswebsite/pull/989) - Replace home page banner image
* [jamulus/jamuluswebsite#982](https://github.com/jamulussoftware/jamuluswebsite/pull/982) - Update iOS page with USBC and further information
* [jamulus/jamuluswebsite#994](https://github.com/jamulussoftware/jamuluswebsite/pull/994) - [Add language selection dropdown](https://github.com/jamulussoftware/jamuluswebsite/issues/992)

**Comment by pljones** [comment]
> - [ ] Check the needs documentation label for any outstanding PRs flagged for this release and remove that label if done.

* #3159 Fixes #1421: Add "delete server" button to connect dialog
* #3260 #3233 UI: Common method to delete server and custom directory entries
* #3305 Convert delete buttons to from QPushButton to QToolButton
* #3299 Add debug_and_release to use separate build dirs

The above are tagged for needing documentation update (I just added #3305).

Do any of the dependency updates need website documentation updates?

**Comment by ann0see** [comment]
> Is this still needed (the .ts files) or is that what's now being done automatically?

I don't think that's done automatically 

> Is this still the correct process?

In theory yes, but since he's no longer that active, we need to find an alternative. E.g @danryu (?)

**Comment by softins** [comment]
> > Is this still needed (the .ts files) or is that what's now being done automatically?
> 
> I don't think that's done automatically

Correct. The `.ts` files need to get updated directly on upstream using `lupdate`, and then the translations need to be checked via weblate.

The part that is now done automatically is the creation of the `.qm` files from the `.ts` files. So there is no longer any need to do a manual `lrelease`, nor to commit `.qm` files.

I have a pending action to update the Release Process on the website to reflect the above, but got sidetracked by difficulties installing Jekyll (not yet resolved). I had originally created `RELEASE-PROCESS.md` in the `jamulus` repo, but it got moved to the website by someone some time ago, unfortunately.

**Comment by softins** [comment]
I've just done the `lupdate` and pushed directly to `main`, so the `.ts` files are ready for the final translation pass for each language. For most languages, there are 19 strings that need translating.

**Comment by softins** [comment]
I've also just removed the `lrelease` step from the checklist above, as it's no longer needed.

**Comment by softins** [comment]
> Is there a template somewhere it also needs removing from?

Just found it in the website, https://github.com/jamulussoftware/jamuluswebsite/blob/release/contribute/en/Release-Process.md

**Comment by ann0see** [comment]
I have the feeling that we should take our time to read through some parts of the website before freezing it - especially the install guides.

**Comment by pljones** [comment]
#3299 -- @softins I've had a look through `COMPILING.md` and I can't see anywhere that would _require_ changes or where it would help by adding anything.  Can you confirm we can live without mentioning the change anywhere?

(I'm mildly interested in what effect the developer sees from making it... but we don't say much about the build process anyway, anyway...)

**Comment by pljones** [comment]
For 
* https://github.com/jamulussoftware/jamulus/pull/3159
* https://github.com/jamulussoftware/jamulus/pull/3260
* https://github.com/jamulussoftware/jamulus/pull/3305

I'm proposing a single new issue be raised in jamulus/jamuluswebsite to get the documentation change done (as all three go together, really).  Before I open it, is anyone already working on this?

**Comment by pljones** [comment]
`Jamulus -h` looks unchanged from r3_10_0:
```
$ git show -s
de60e5eb 2024-07-23 (HEAD -> main, upstream/main, origin/main) Update .ts files ready for 3.11.0 translations [Tony Mountifield]
$ ./Jamulus -h > de60e5eb
```

```
$ git show -s
9de9ad70 2023-09-03 (HEAD, tag: r3_10_0, tag: latest) Update version to 3.10.0 for release [ann0see]
$ ./Jamulus -h > r3_10_0
```

```
$ diff -du r3_10_0 de60e5eb
$ $ echo $?
0
```

**Comment by ann0see** [comment]
> is anyone already working on this?

No. I don't think so.

**Comment by pljones** [comment]
I've opened #3316 with a draft change log.

**Comment by ann0see** [comment]
@danryu @emlynmac any updates on signing for macOS?

**Comment by pljones** [comment]
https://github.com/jamulussoftware/jamuluswebsite/issues/1002 raised for the "delete server" button changes.

**Comment by pljones** [comment]
> I've opened #3316 with a draft change log.

I had `CHANGELOG: SKIP` on #3116 -- but the script has tried to write a ChangeLog entry... @softins any ideas?  Did I get the notation wrong?

**Comment by pljones** [comment]
Just updated the ChangeLog to have 3.11.0 as the next release on jamulus/jamulus/main.

**Comment by pljones** [comment]
#3318 add to 3.12.0 to fix handling of the change log tag other than at the start of a line (e.g. inside quoted blocks).

**Comment by pljones** [comment]
#3319 added to 3.12.0 with some feature requests to improve sorting and grouping in the change log helper.

**Comment by pljones** [comment]
#3320 added for a process documentation tweak for something I missed: checking the header of the change log to see the version is what we're planning to release.

**Comment by softins** [comment]
> #3299 -- @softins I've had a look through `COMPILING.md` and I can't see anywhere that would _require_ changes or where it would help by adding anything. Can you confirm we can live without mentioning the change anywhere?
> 

I've read through it, and I can't see anything that needs to change as a result of #3299. We might want to mention that because of  #3288, the minimum supported version of Qt is now 5.12, so it would be tricky to build on an older Linux distro that only provides an earlier version. Also, I noticed that in the Windows section, it recommends Qt 5.15.2, although our CI Windows builds are now using Qt 6.x.

> (I'm mildly interested in what effect the developer sees from making it... but we don't say much about the build process anyway, anyway...)

Not much effect. The built binary still ends up in the same place in the project root. It's just that the project root is a lot less cluttered with build files such as `moc_*.cpp` and `*.o`, which are now placed in `release/` instead of the root.

**Comment by danryu** [comment]
> @danryu @emlynmac any updates on signing for macOS?

Unfortunately the account for Koord is no longer available, and I don't have a personal developer account yet.
If nobody else has an Apple developer account, perhaps a new Jamulus developer account needs setting up?

**Comment by ann0see** [comment]
> perhaps a new Jamulus developer account needs setting up

I believe this would be the way to go. I'm thinking about getting a personal one for myself which we can use to sign Jamulus. An organization wide one needs DUNS numbers and seems to imply a lot of hurdles.

**Comment by ann0see** [comment]
Heads up: @ignotus666 I force pushed to the release branch since updating via github web ui mistakenly merged the next-release branch to release pre release but luckily didn't change the website. I think that everything we need is on next-release.

**Comment by ignotus666** [comment]
@ann0see yes, I saw that - the failing workflows stop it from generating the website.

> I think that everything we need is on next-release.

Yep, I think so.

**Comment by ann0see** [comment]
> the failing workflows stop it from generating the website.

luckily...

**Comment by pljones** [comment]
> > #3299 -- @softins I've had a look through `COMPILING.md` and I can't see anywhere that would _require_ changes or where it would help by adding anything. Can you confirm we can live without mentioning the change anywhere?
> 
> I've read through it, and I can't see anything that needs to change as a result of #3299. We might want to mention that because of #3288, the minimum supported version of Qt is now 5.12, so it would be tricky to build on an older Linux distro that only provides an earlier version. Also, I noticed that in the Windows section, it recommends Qt 5.15.2, although our CI Windows builds are now using Qt 6.x.
> 
> > (I'm mildly interested in what effect the developer sees from making it... but we don't say much about the build process anyway, anyway...)
> 
> Not much effect. The built binary still ends up in the same place in the project root. It's just that the project root is a lot less cluttered with build files such as `moc_*.cpp` and `*.o`, which are now placed in `release/` instead of the root.

Ah, #3288 also wasn't being tracked.

**Comment by pljones** [comment]
I'll let someone else do the broken link check.  Ubuntu 22.04LTS says
```
$ _po4a-tools/po4a-create-all-targets.sh
Error: po4a v0.66. is installed
po4a v0.68 or higher is required.
```
after a fresh install of `po4a`.

**Comment by pljones** [comment]
Can someone also confirm the translation processes under both "Start Website translations" and "Start App translations" are current, please.

**Comment by ann0see** [comment]
You could check https://github.com/jamulussoftware/assets for a po4a deb.

Otherwise I'll check it later tonight

**Comment by ann0see** [comment]
https://github.com/jamulussoftware/jamulus/commit/3ac940a97c9e6204d4d1683e7f852482357584b0 shows that the app should be ready.

**Comment by trebmuh** [comment]
Hi all.
For information (not sure if it's relevant here), I just made an update for the French translation on Weblate.

**Comment by ann0see** [comment]
No broken links found.

**Comment by ann0see** [comment]
@jamulussoftware/translators I declare the next-release branch as frozen now. We may need to add some updates past release, but try to do our best to not needing re translations!

You can start translating the website and app on weblate now.

**Comment by ann0see** [comment]
Also, we should really check who is still a website/app translator in the upcoming release and then update the create-translation-issues script with the active ones.

**Comment by pljones** [comment]
For screenshots, the two updates I can see are these:
```
$ git diff --name-only release next-release assets/img/en-screenshots/
assets/img/en-screenshots/connection-setup-window.inc
assets/img/en-screenshots/settings-advanced.inc
```
Should the issues for translators just have these two to be replaced?

**Comment by pljones** [comment]
Beta release tagged.
```
cb446699 2024-08-07 (HEAD -> main, origin/main) Revert to dev status [Peter L Jones]
3593ba56 2024-08-07 (tag: r3_11_0beta1) Update version to 3.11.0beta1 [Peter L Jones]
```

**Comment by pljones** [comment]
https://github.com/jamulussoftware/jamuluswebsite/issues/1024 raised for 3.12.0 to fix a small website issue -- given we've frozen the translations and it's been wrong for ages, there's no rush to fix it for 3.11.0.

**Comment by ann0see** [comment]
I'm trying to get a personal Apple Developer account soon. The process seems a hassle but we'll get there.

**Comment by ann0see** [comment]
I think in the next few weeks it'll be there (I already enrolled and paid but am not sure how much time the setup needs)

**Comment by ann0see** [comment]
Once the build is finished maybe we'll have a signed DMG: https://github.com/ann0see/jamulussign/releases/tag/r3_11_0beta1

**Comment by ann0see** [comment]
Notsituation will probably not work yet.

**Comment by ann0see** [comment]
@emlynmac can you transfer the app to me such that I can have the old bundle identifier?

**Comment by emlynmac** [comment]
> @emlynmac can you transfer the app to me such that I can have the old bundle identifier?
Please send me details and I can transfer.

**Comment by ann0see** [comment]
Done

**Comment by ann0see** [comment]
Maybe it suffices to remove the bundle ID: https://stackoverflow.com/questions/55254333/how-to-move-an-app-with-the-same-bundle-identifier-to-another-apple-developer-ac

**Comment by emlynmac** [comment]
> Maybe it suffices to remove the bundle ID: https://stackoverflow.com/questions/55254333/how-to-move-an-app-with-the-same-bundle-identifier-to-another-apple-developer-ac

So, 
1. Can't transfer as a release was not made.
2. Can't reuse the bundle ID as that is not allowed, I sent you the details by email.

Check your email for details of what to do next.

**Comment by ann0see** [comment]
Which means that users will need to migrate their settings. Not too bad, but anyway.

**Comment by ann0see** [comment]
Ok. Notarization has been pending for a few days. Probably apple does an in depth review for first timers. I'd say we continue with an unsigned beta 1 and focus on releasing beta 2 with translations.

**Comment by ann0see** [comment]
Regarding the bundle ID topic. io.jamulus.Jamulus is not available anymore, io.jamulussoftware.jamulus isn't either. Any suggestions?

**Comment by gilgongo** [comment]
Could we borrow @corrados user name? His SF project is "llcon" too...

**Comment by ann0see** [comment]
Possibly, but I'd prefer not to. I just tried com.jamulus.jamulus and it seems to work

**Comment by ann0see** [comment]
Also @gilgongo could you please check if the element -> DC bridge is working? I don't get any messages

**Comment by pljones** [comment]
> Possibly, but I'd prefer not to. I just tried com.jamulus.jamulus and it seems to work

I'd rather avoid the `com.` prefix.  Do we know where the `io.jamulus` prefix is?  Is this with @emlynmac?

**Comment by ann0see** [comment]
Yes. The io is with Emlyn and unless we release Jamulus via his account on the App Store we cannot migrate the ID. I'm not sure if he wants to release it on the App Store – I'd be fine, but it seems as if that's a bigger hurdle.

**Comment by ann0see** [comment]
I'd go with either org.* or app.*

**Comment by ann0see** [comment]
As beta, see https://github.com/ann0see/jamulussign/releases/tag/r3_11_0beta1sign4 It's signed.

**Comment by ann0see** [comment]
I'd vote for app.* @pljones @softins are you ok with that?

**Comment by ann0see** [comment]
Ok. I'm doing the respective changes then.

**Comment by ann0see** [comment]
Since the translation deadline has been over for some days, I'd suggest to review and merge the website and app PRs tomorrow, then tag a beta or even RC1.

**Comment by ann0see** [comment]
Due to the signing + bundle ID changes on macOS, we may want to have a beta2, not a rc.

**Comment by pljones** [comment]
I was also going to suggest a beta2 in the light of the PNG changes, just in case something went wrong with that.

**Comment by pljones** [comment]
@ann0see I've attached the signed mac builds to the beta1 release...  I've left the unsigned ones there for now, too.

**Comment by ann0see** [comment]
Ok. They aren't officially speaking the "real" beta1 builds since they have another version identifier. They are built on the same code base nevertheless.

**Comment by pljones** [comment]
> Ok. They aren't officially speaking the "real" beta1 builds since they have another version identifier. They are built on the same code base nevertheless.

So long as it was against the tag, that's what matters - i.e. `./Jamulus --version` and "Help->About Jamulus" say the right thing, i.e.
> Jamulus, Version 3.11.0beta

**Comment by ann0see** [comment]
No. It does not. But I think it doesn't matter too much as we should get out beta2 asap

**Comment by ann0see** [comment]
Tagged https://github.com/jamulussoftware/jamulus/actions/runs/10565063718 beta2

**Comment by pljones** [comment]
Rock, Jazz, Classical/Folk, Choral/Barbershop Directories cut over to beta2.

My "2", "6" and "8" Directories, my public server is on main (as is my private server, currently, as it's code is frozen, too).

**Comment by pljones** [comment]
I'd like to get the RC by no later than 8th September, then bump to release on 15th, assuming it's all clear.

**Comment by pljones** [comment]
> - [x] Finish App translations
>   - [x] Review translation PRs according to [release process checklist](https://jamulus.io/contribute/Release-Process#3-update-the-ts-files-returned-by-translators)
>   - [x] Wait for all PRs to be merged (missing translations will revert to English automatically).
>   - [ ] Check for conflicting accelerator keys (see tools/checkkeys.pl)
>   - [ ] [Check for broken links](https://github.com/jamulussoftware/jamuluswebsite/blob/release/contribute/en/Release-Process.md#website-check-links)

"Check for broken links" is a website check -- needs removing from the "Finish App translations".

**Comment by ann0see** [comment]
Accelerator keys and some change log cleanup needs to happen still.

**Comment by softins** [comment]
I have been intending to check through the accelerator keys this week. Hopefully tomorrow.

**Comment by softins** [comment]
I've checked the accelerator keys in all languages, and there were no conflicts to resolve. I'll mark the task as done in the checklist.

**Comment by pljones** [comment]
Anything else before the RC?  Just the ChangeLog (which needs to include the translators and be trimmed down so it's not flooded with entries for version bumps)?

**Comment by ann0see** [comment]
@pljones Yes, I think so

**Comment by ann0see** [comment]
@pljones there seems to be an issue with building the RC: https://github.com/jamulussoftware/jamulus/actions/runs/10854740671/job/30125832570

**Comment by pljones** [comment]
```
 ./.github/autobuild/extractVersionChangelog.pl ChangeLog 3.10.0rc1
```
Why does it do that?

**Comment by pljones** [comment]
Argh, typo.

**Comment by pljones** [comment]
Better
https://github.com/jamulussoftware/jamulus/actions/runs/10855006639

**Comment by pljones** [comment]
Announcement posted: https://github.com/orgs/jamulussoftware/discussions/3371
@ann0see please edit once you've got the notarised build in place.

**Comment by pljones** [comment]
> - [x] Update the Changelog
> - [x] Tag a beta release

> - [x] Update the Changelog
> - [x] Tag a release candidate

> - [x] Update the version number in Jamulus.pro and add the release date to the Changelog header and commit
> - [x] Update the Changelog
> - [x] Tag this commit as r3_y_z

Why for the final release is the build broken up into three steps, with the "Update the Changelog" _after_ actually pushing the release build and tagging it and then _before_ trying to tag it again?  This needs updating.

**Comment by pljones** [comment]
Release 3.11.0 is out now.  I've deployed to the update02 server, SourceForge default downloads are up to date and notices are up on Facebook and GitHub.  Summarising the release check list points outstanding:

> - [ ] Assign this issue to the release shepherd who is in charge of managing this checklist.

Looks like we're able to live without this formally being assigned and I guess we may as well remove it from the check list.

> - [ ] Check if the list of translators in tools/create-translation-issues.sh.
Make sure issue text is up-to-date.
Add any URLs that will need localisation into the "New/Changed screenshots" section.

I think this needs looking into.  We don't seem to know how to find out who is still involved other than by hoping they are.
- How do we keep the translators engaged during the development process?  Is this another point in favour of small frequent releases (so small amounts of work more often to retain engagement)?

> - [ ] Check for broken links

Hopefully this should now work but I never went back and checked...

> - [ ] Trigger the update notification by updating both Update Check Servers with the new version (@pljones for update02, email corrados for update01)

I was _very_ late getting in touch with Volker, most because we'd not been particularly firm on a release date.  We _do_ always have the option of repointing the update01 DNS name until Volker deploys a release, anyway.

> - [ ] Update this template in https://jamulus.io/contribute/Release-Process with any improvements if needed.

I've raised at least one issue for changing things and tagged it for 3.12.0.  If anyone else has anything, please raise it as an issue.  (I just moved one of mine to jamuluswebsite, which is where it should be raised.)

> - [ ] Determine if a release retrospective is needed, create on Discussions if required

Anything else anyone wants?  Please raise it on Discussions, if so.

Thank you, all!
